server/routes.go

```diff
@@ -657,6 +657,28 @@ func (s *Server) ShowModelHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, resp)
 }
 
+func (s *Server) RetrieveModelHandler(c *gin.Context) {
```

Comment:
Let's see if we can keep this in the `openai` package

Reply:
I'm not really sure how to do that without creating an import cycle between the `openai` and `server` package... let me know if there's an easy way around that. server contains `*Server` and `ParseNamedManifest`